setup.py

```diff
@@ -79,7 +79,7 @@
     ],
     extras_require={
         ":python_version<'3.0'": ['pylint==1.9.2', 'futures'],
-        ":python_version>='3.0'": ['pylint==2.3.0']
+        ":python_version>='3.0'": ['pylint==2.3.0', 'astroid==2.4.2']   # pylint 2.3 doesn't support astroid 2.5
```

Comment:
Using condition like `":python_version>='3.0'"` in `extras_require` has been dropped. It is not documented anymore:

https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#optional-dependencies